README.md

```diff
@@ -1541,20 +1419,20 @@ We'll use:
             {:name "green" :message "green makes it go faster"}]))
 ```
 
-Now ```chat-messages``` is pointing to the ```atom``` protecting our vector
+Now `chat-messages` is pointing to the `atom` protecting our vector
 of hashes.
 
-Because ```chat-messages``` is pointing to the ```atom```, we can't simply
-```map``` over it in ```generate-message-view```.  Now we have to tell Clojure
-that we want to generate HTML for the content of the atom.  This allows Clojure
-to ensure the messages are always read in a consistent state, even though
-something could be modifiying them.
+Because `chat-messages` is pointing to the `atom`, we can't simply
+`map` over it in `generate-message-view`.  Now, we have to tell
+Clojure that we want to generate HTML for the contents of the atom.  This
+allows Clojure to ensure the messages are always read in a consistent
+state, even though something could be modifiying them.
 
```

Comment:
Not a nitpick at all - I went back and forth on this one.  Contents in English is usually considered a plurale tantum, meaning that the plural is used for singular situations as well (like "pants").  In the case of "contents", the -s marker denotes countability, and we use "content" for uncountable nouns ("the content of their character").

But, in this case, I think it is important to emphasize that atoms hold a singular value.  It might be worth its own sentence.

Thanks!